installation/scripts/installer.sh

```diff
@@ -92,13 +91,13 @@ if [ ${LOCAL} ]; then
     COMBO_YAML=$(sed 's/\.minikubeIP: .*/\.minikubeIP: '"${MINIKUBE_IP}"'/g' <<<"$COMBO_YAML")
 fi
 
-kubectl apply -f - <<<"$COMBO_YAML"
-
 echo -e "\nConfiguring sub-components"
 bash ${CURRENT_DIR}/configure-components.sh
 
-echo -e "\nGetting Helm secrets"
-bash ${CURRENT_DIR}/tiller-tls.sh
+echo -e "\nApplying combo yaml"
```

Comment:
I would put this section at the end (after Helm certificates), and log just: "Starting Kyma Installation" or something like that.